src/components/select/index.tsx

```diff
@@ -0,0 +1,135 @@
+import React from 'react';
+import './style.less'
+import DatePicker from "react-datepicker";
+import "react-datepicker/dist/react-datepicker.css";
+import { getMovies } from '../../services/movieService';
+import { MovieDay, Movie, CinemaMovieSchedule } from '../../models/movieDay';
+import { isSameDay } from '../../services/calendarService';
+import MovieList from '../movie-list';
+
+
+interface State {
+    cities: Array<string>;
+    selectedCity: string;
+    selectedDate: Date;
+    showMovies: boolean;
+    movies: Array<Movie>;
+}
+
+interface Props { }
+
+
+class Select extends React.Component<Props, State> {
```

Comment:
1. Name 'Select' isn't quite clear. 'SelectCityAndDate' could be more appropriate. 
2. Empty props do not need to have their own interface. It can then be simply typed as follows:
```suggestion
class Select extends React.Component<Props, {}> {
```